Description: Drop intel from the autoloaded drivers for i830, i845g, i855
 .
 These remain too unstable with the intel DDX.
Author: Christopher James Halse Rogers <christohper.halse.rogers@canonical.com>

Index: xorg-server/hw/xfree86/common/xf86pciBus.c
===================================================================
--- xorg-server.orig/hw/xfree86/common/xf86pciBus.c	2010-09-13 19:54:39.330148002 +1000
+++ xorg-server/hw/xfree86/common/xf86pciBus.c	2010-09-13 20:54:08.000000000 +1000
@@ -1112,6 +1112,12 @@
 		driverList[0] = "i740";
             } else if (dev->device_id == 0x8108) {
                 break; /* "hooray" for poulsbo */
+	    } else if (dev->device_id == 0x3577) {
+		break; /* Let i830 fall back to fbdev / vesa */
+	    } else if (dev->device_id == 0x2562) {
+		break; /* Let i845g fall back to fbdev / vesa */
+	    } else if (dev->device_id == 0x3582) {
+		break; /* Let i855 fall back to fbdev / vesa */
 	    } else {
 		driverList[0] = "intel";
 	    }
